pkg/fission-cli/util/util.go

```diff
@@ -149,20 +149,48 @@ func GetVersion(ctx context.Context, input cli.Input, cmdClient cmd.Client) info
 }
 
 func GetServerInfo(input cli.Input, cmdClient cmd.Client) *info.ServerInfo {
-	serverUrl, err := GetServerURL(input, cmdClient)
+
+	serverURL, err := getRouterURL(input.Context(), cmdClient)
+	if err != nil {
+		console.Warn("could not connect to server")
+		return nil
+	}
+	// make request
+	resp, err := http.Get(serverURL.String() + "/_version")
 	if err != nil {
-		return &info.ServerInfo{}
+		console.Warn("could not get data from server")
+		return nil
 	}
-	restClient := rest.NewRESTClient(serverUrl)
-	client := client.MakeClientset(restClient)
 
-	serverInfo, err := client.V1().Misc().ServerInfo()
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		msg := fmt.Sprintf("HTTP error %v", resp.StatusCode)
+		console.Warn(msg)
+		return nil
+	}
+
+	var serverInfo info.ServerInfo
```

Comment:
Since we returned empty valid server info earlier, should we replace it?
This may break functions calling this function as they try to dereference nil pointer.

Should we move this definition to the top and return if any error ?

```
	var serverInfo info.ServerInfo
```